src/battle_anim_mons.c

```diff
@@ -966,7 +966,7 @@ void sub_80A6D60(struct BattleAnimBgData *unk, const void *src, u32 arg2)
     CopyBgTilemapBufferToVram(unk->bgId);
 }
 
-u8 sub_80A6D94(void)
+u8 GetPaletteOffset(void)
```

Comment:
`GetPaletteOffset()` needs a better name.  What is the offset for?  The battle background (etc.?)?

Reply:
This functions returns 1 for contest, else 2. According to `LoadCompressedPalette` it's a palette offset...
Welp, when I see the use in `LoadMoveBg`, I think `GetPaletteOffset` is kind of pointless anyway, I don' tknow how to label that

Reply:
I investigated this.  Its the palette number for the battle background.  So something like `GetBattleBgPaletteNum()` is fine.